DEV_ENV.md

```diff
@@ -1,12 +1,11 @@
 # Local Development Environment
 
-## Development quickstart
+## Install Docker
 
 1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`. If you have a Mac, [install Docker Desktop](https://www.docker.com/products/docker-desktop) and open it so it's running on your machine. Note: If you have a Mac M1 (arm64 CPU), follow instructions in [`docker-compose.yml`](docker-compose.yml) under `oidc` service, for manually building the oidc-server-mock images for the M1 architecture.
```

Comment:
can you add a line about setting up docker with the CZI enterprise account? something like: "Go to https://hub.docker.com/ and sign in with SSO, which should add yourself to the CZI org automatically. Verify that you see yourself here: https://hub.docker.com/orgs/chanzuckerberg/members"

Reply:
This is a public repo, so I don't think this is a good idea.

Reply:
:homer-disappears: